specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2022-02-01-preview/scanningService.json

```diff
@@ -0,0 +1,7565 @@
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "classificationRuleName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ClassificationRuleList"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ClassificationRules_ListVersionsByClassificationRuleName": {
+            "$ref": "./examples/ClassificationRules_ListVersionsByClassificationRuleName.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/classificationrules/{classificationRuleName}/versions/{classificationRuleVersion}/:tag": {
+      "post": {
+        "tags": [
+          "ClassificationRules"
+        ],
+        "description": "Sets Classification Action on a specific classification rule version.",
+        "operationId": "ClassificationRules_TagClassificationVersion",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "classificationRuleName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "classificationRuleVersion",
+            "required": true,
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "in": "query",
+            "name": "action",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "Keep",
+              "Delete"
+            ],
+            "x-ms-enum": {
+              "name": "ClassificationAction",
+              "modelAsString": true
+            }
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/OperationResponse"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ClassificationRules_TagClassificationVersion": {
+            "$ref": "./examples/ClassificationRules_TagClassificationVersion.json"
+          }
```

Comment:
Operations with a 202 response should specify `x-ms-long-running-operation: true`.